### PR TITLE
chore: add react-outputtarget

### PR DESCRIPTION
**Pull request by JCofman**
Closes https://github.com/inovex/elements/issues/404

## Proposed Changes

- adds official react-output-target implementation by stencil to generate react components

## Things to check

- [ ] Does the change need to be documented?
- [ ] Does any existing example code needs to be updated?
- [ ] Is the change properly tested?
- [ ] Is it helpful to provide another example to demonstrate the new feature?
- [ ] Are there other code lines that need to be modified?

### CODE REVIEW

**Comment by janivo** [comment]
Almost, the name of the branch has to start with `elements-` 😅

File: packages/elements-react/package.json [hunk 1]
```diff
@@ -59,18 +59,14 @@
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-react": "^7.22.0",
     "np": "^3.1.0",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-router": "^4.3.1",
-    "react-router-dom": "^4.3.1",
+    "react": "^17.0.2",
```
Comment (silentHoo) [inline packages/elements-react/package.json:62]: There's a breaking change in here we need to document: The React version changes from 16 -> 17 which breaks a few things for the consuming applications.

**Comment by JCofman** [comment]
Ok, I am actually not sure what I do wrong I have created several new branches with specific names starting `elements-`. Is there a reason we have introduced the rule and where can I find it 😅

**Comment by janivo** [comment]
see #537

**Pull request closed without merging**